src/PgRvn.Server/PgConfig.cs

```diff
@@ -0,0 +1,3288 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace PgRvn.Server
+{
+    public static class PgConfig
+    {
+        // TODO: Customize these
+        // TODO: Support changing these after startup by the user
+        public static readonly Dictionary<string, string> ParameterStatusList = new(StringComparer.OrdinalIgnoreCase)
+        {
+            ["client_encoding"] = "UTF8",
+            ["server_encoding"] = "UTF8", // Cannot be modified after startup
+            ["server_version"] = "13.3", // Cannot be modified after startup
+            ["application_name"] = "",
+            ["DataStyle"] = "ISO, DMY",
+            ["integer_datetimes"] = "on", // Cannot be modified after startup
+            ["IntervalStyle"] = "postgres",
+            ["is_superuser"] = "on",
+            ["session_authorization"] = "postgres",
+            ["standard_conforming_strings"] = "on",
+            ["TimeZone"] = "UTC",
+        };
+
+        public static byte[] TrueBuffer = { 1 }, FalseBuffer = { 0 };
+
+        public static readonly PgTable NpgsqlInitialQueryResponse = new()
```

Comment:
This is simply impossible to maintain or work with.
Better to actually keep that as data directly.

Reply:
Done